src/styles/_layout.scss

```diff
@@ -0,0 +1,86 @@
+/*****************************************************************************
```

Comment:
This is for sure going to collide with https://github.com/nasa/openmct/pull/2567/files#diff-93a3ad37a45a7bbfd40fe7b0162d077f

I assume that there are substantive changes here, and this has not just been moved into `_layout.scss`? If so, I will merge #2567 first, and we can put the styles below into [browse-bar.scss](https://github.com/nasa/openmct/pull/2567/files#diff-93a3ad37a45a7bbfd40fe7b0162d077f)

Reply:
Will be resolved by removing browse-bar.scss in #2567